tox.ini

```diff
@@ -65,6 +65,8 @@ allowlist_externals = ansible-playbook
 commands =
     mypy --txt-report ./{[base]mypy_tmp_dir} ./{[base]pkg_name} ./tests ./share
 
+[testenv:venv]
```

Comment:
@pabelanger why not just use `tox --devenv` for this? No need for a separate toxenv.

Reply:
Mostly because this has been the only way to do it until https://github.com/tox-dev/tox/pull/1644.  I'll have to see how the command works. But for now, we can close this PR, I've made the changes in another repo.

Reply:
sorry: https://github.com/tox-dev/tox/pull/1326